odl/test/largescale/solvers/nonsmooth/default_functionals_slow_test.py

```diff
@@ -176,10 +176,24 @@ def test_proximal_defintion(functional, stepsize):
 
 
 def convex_conj_objective(functional, x, y):
-    """CObjective function of the convex conjugate problem."""
+    """Objective function of the convex conjugate problem."""
     return x.inner(y) - functional(x)
 
 
+def func_convex_conj_has_call(functional):
+    """Helper to determine whether a convex conjugate can be called."""
+    f_cconj = functional.convex_conj
+    if isinstance(f_cconj, FunctionalDefaultConvexConjugate):
+        return False
+    if (
```

Comment:
Write this as 

```
if (a or
    (b and
     c)):
```

Reply:
It's only a and b in this case, and that results in the ugly extra indent of the second line (that I've always disliked):
```python
if (isinstance(f_cconj, odl.solvers.FunctionalTranslation) and
        isinstance(f_cconj.functional, FunctionalDefaultConvexConjugate)):
    # Stuff
```
I find this version much more readable since the statements line up.

Reply:
An alternative would be to start on the first line and just put the last `):` on an extra line.

Reply:
I went with the current version, but without the line break after the opening parenthesis. Hence this:
```python
if (isinstance(f_cconj, odl.solvers.FunctionalTranslation) and
    isinstance(f_cconj.functional, FunctionalDefaultConvexConjugate)
):
    # Stuff
```